Mathlib/Algebra/MvPolynomial/Monad.lean

```diff
@@ -268,8 +268,7 @@ theorem aeval_bind₂ [Algebra S T] (f : σ → T) (g : R →+* MvPolynomial σ
     aeval f (bind₂ g φ) = eval₂Hom ((↑(aeval f : _ →ₐ[S] _) : _ →+* _).comp g) f φ :=
   eval₂Hom_bind₂ _ _ _ _
 
-theorem eval₂Hom_C_left (f : σ → MvPolynomial τ R) : eval₂Hom C f = bind₁ f :=
-  rfl
+@[deprecated (since := "2024-07-28")] alias eval₂Hom_C_left := eval₂Hom_C_eq_bind₁
```

Comment:
I would advocate keeping this name in (but making it an alias).
It adds discoverability. And it also means that we can keep this file "complete": covering all the monad laws for these mv-polynomial monads.

Reply:
I don't quite follow - do you mean to keep it as an alias but not deprecate it (and revert the WittVector changes)?

Reply:
Yes, exactly. I would like to keep both names in the library, because I think both make sense.

I don't care too much which version of the name ends up being used in certain parts of the library. But I certainly thought about these operations and lemmas in terms of the monad laws, while working on the Witt vector development.